src/FormBuilder/resources/js/form-builder/src/components/OptionsPanel/index.tsx

```diff
@@ -28,15 +42,14 @@ export default function Options({currency, multiple, selectable = true, options,
                 <BaseControl id={'give'}>
                     <OptionsHeader handleAddOption={handleAddOption} />
                     <OptionsList
-                        {...{
-                            currency,
-                            options,
-                            showValues,
-                            multiple,
-                            selectable,
-                            setOptions,
-                            defaultControlsTooltip,
-                        }}
+                        currency={currency}
+                        options={options}
```

Comment:
I don't know if this is related, but I noticed the donor name honorific options is being displayed as numbers:

<img width="321" alt="Screenshot 2023-11-02 at 4 19 04 PM" src="https://github.com/impress-org/givewp/assets/10138447/c4656786-3cc9-4f76-857d-88cbfb4973dd">

Reply:
Was fixed in #7088